src/libraries/Microsoft.PowerFx.Core/Utils/TextFormatUtils.cs

```diff
@@ -42,5 +170,21 @@ public static bool IsValidFormatArg(string formatArg, out bool hasDateTimeFmt, o
 
             return true;
         }
+
+        private static bool TryGetCulture(string name, out CultureInfo value)
```

Comment:
>private static bool TryGetCulture(string name, out CultureInfo value) [](http://example.com/codeflow?start=8&length=69)

There is a similar block at [https://github.com/microsoft/Power-Fx/blob/f660b11723af19cb94f3ea67f6b76257a7811c6f/src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryDate.cs#L557](https://github.com/microsoft/Power-Fx/blob/f660b11723af19cb94f3ea67f6b76257a7811c6f/src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryDate.cs#L557).
Is is possible you can remove it form there and call this nice helper?

Reply:
Remove there and update all references.